main.nf

```diff
@@ -3694,6 +3755,8 @@ def nfcoreHeader() {
     c_reset  = params.monochrome_logs ? '' : "\033[0m";
     c_white  = params.monochrome_logs ? '' : "\033[0;37m";
     c_yellow = params.monochrome_logs ? '' : "\033[0;33m";
+    
+    return "--"
```

Comment:
no sure why you added that

Reply:
Because I could not see the debug messages. Will be restored eventually :)